CHANGELOG.rst

```diff
@@ -7,6 +7,13 @@ Change Log
 ----------
 
 
+0.7.4
```

Comment:
Should definitely be major version IMO

Reply:
Major as in '1.0.0'? (Or just '0.8.0')?

Reply:
@willronchetti, not sure how you feel about this but I might say 1.0 just so we start using proper semantic versioning?